### DISCUSSION TITLE
i need help

### DISCUSSION THREAD

**Discussion by Grigriye** [Techokami/SonicWorldsNext#107] [General]
Hello, i am very new to game making and i wanted to make a sonic game, i found this because it was on a beginner engine so i thought i could use it but i dont really understand the basics of it all. can somebody please help? i really want to make a sonic game and this seems like the best way to do so.

**Comment by DimensionWarped** [comment]
I'd strongly recommend learning the basics of Godot before you try to tackle this particular engine. Unfortunately, we can't really get you off the ground before that point. That said, if you have more specific questions one of us can at least try to answer them. You might consider starting with a single topic like "[How do I make a new level](https://github.com/Techokami/SonicWorldsNext/wiki/Levels)" or "What would I need to do to make [x object] from [y sonic game and level]".

If you stick with it and ask good pointed questions, we can turn those into pages of the wiki so that people have an easier time getting started in the future.

**Reply by Grigriye** [reply]
ahh alright, thanks alot this single comment has helped so much, thanks alot for that
